ui/app/AppLayouts/Wallet/components/TokenSettingsModalContent.qml

```diff
@@ -109,12 +123,13 @@ Item {
         anchors.bottom: parent.bottom
 
 
-        Item {
+        Rectangle {
```

Comment:
Again, I don't think this needs to be a Rectangle

Reply:
Also a leftover!